dll/cpl/mmsys/sounds.c

```diff
@@ -1140,9 +1228,19 @@ SoundsDlgProc(HWND hwndDlg,
 
             pGlobalData->NumWavOut = waveOutGetNumDevs();
 
-            SendMessageW(GetDlgItem(hwndDlg, IDC_PLAY_SOUND),
-                         BM_SETIMAGE, (WPARAM)IMAGE_ICON,
-                         (LPARAM)(HANDLE)LoadIconW(hApplet, MAKEINTRESOURCEW(IDI_PLAY_ICON)));
+            pGlobalData->hPlayIcon = LoadImageW(hApplet,
+                                                MAKEINTRESOURCEW(IDI_PLAY_ICON),
+                                                IMAGE_ICON,
+                                                32,
```

Comment:
hardcoded 32, or `GetSystemMetrics(SM_CXICON)` (and similar for the Y dimension) ?

Reply:
The ico file has only 32x32 icon, that's why it's hardcoded 32.